yarn-project/noir-contracts/package.json

```diff
@@ -4,7 +4,7 @@
   "type": "module",
   "exports": {
     "./artifacts": "./dest/artifacts/index.js",
-    "./types": "./dest/types/index.js"
+    "./types": "./dest/index.d.ts"
```

Comment:
I think `types` needs to go as a top-level key to make it pass the check.

Reply:
Can you do the fix? not sure what you mean